Makefile

```diff
@@ -1,2 +1,2 @@
 all:
-	gcc gameboy.c platform.c sound.c -lSDL -I /usr/include/SDL -o gameboy -Os
+	gcc gameboy.c platform.c sound.c -lSDL -I /usr/include/SDL2 -o gameboy -Os
```

Comment:
You are still linking to the old version of SDL. `-lSDL` needs to be changed to `-lSDL2`.
Also remove the space after `-I` because currently the folder isn't being included (my epic coding skills of three years ago). So `-I/usr/include/SDL2`.

Once that's done, running make will show the following errors because this application does not have support for SDL2:
```
> $ make                                                                                                                                                                                                                         [±master ●]
gcc gameboy.c platform.c sound.c -lSDL2 -I/usr/include/SDL2 -o gameboy -Os
/tmp/ccOZGkmi.o: In function `main':
platform.c:(.text.startup+0x13c): undefined reference to `SDL_SetVideoMode'
platform.c:(.text.startup+0x14b): undefined reference to `SDL_WM_SetCaption'
platform.c:(.text.startup+0x16b): undefined reference to `SDL_WM_SetCaption'
platform.c:(.text.startup+0x593): undefined reference to `SDL_Flip'
platform.c:(.text.startup+0x60c): undefined reference to `SDL_WM_SetCaption'
collect2: error: ld returned 1 exit status
make: *** [Makefile:2: all] Error 1
```